CMakeLists.txt

```
@@ -18,6 +18,10 @@ if(NOT CMAKE_OSX_DEPLOYMENT_TARGET)
         FORCE)
 endif()
 
+if(VCPKG_TARGET_ANDROID)
+    include("cmake/VcpkgAndroid.cmake")
```

Nit: quoting isn't necessary here:
```suggestion
    include(cmake/VcpkgAndroid.cmake)
```